src/Preferences/AdvancedPreferencesViewController.m

```diff
@@ -65,7 +65,8 @@ - (NSString *)toolbarItemLabel
  */
 -(IBAction)showAdvancedHelp:(id)sender
 {
-    GotoHelpPage((CFStringRef)@"advanced.html", NULL);
+    NSString *helpBook = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleHelpBookName"];
+    [[NSHelpManager sharedHelpManager] openHelpAnchor:@"AdvancedSettingsSection" inBook:helpBook];
```

Comment:
I haven't looked in to it properly, but according to the Apple documentation, NSHelpManager's openHelpAnchor is a wrapper for the old methods. I wonder if it still needs carbon?